CONTRIBUTING.md

```diff
@@ -23,3 +23,37 @@ use GitHub pull requests for this purpose. Consult [GitHub Help] for more
 information on using pull requests.
 
 [GitHub Help]: https://help.github.com/articles/about-pull-requests/
+
+## How to build dagger locally
+
+TODO THESE ARE WIP NOTES
+
+* Go to [http://bazel.build](http://bazel.build) , click on GET BAZEL, install bazel
+* Clone the repository: `git clone git@github.com:google/dagger.git dagger`
+* Install the IntelliJ Bazel plugin: go https://docs.bazel.build/versions/master/ide.html, click on the link, click on Download
+
+ => Run Android Studio > About => Look at the weird version number => download the corresponding plugin.
+* In IntelliJ, click on `Plugins`, `Install plugin from disk...` and select `aswb_bazel.zip`
```

Comment:
I don't understand those steps. Any reason you're not using IntelliJ plugins "portal"? This is what the instructions at the linked page and at https://ij.bazel.build/docs/bazel-plugin.html tell you to do.

Reply:
Searching for "Bazel" in the plugin list did not bring anything up.

So we tried to download the plugin. Turned out none of the plugins available for download are compatible with Android Studio.

The bazel doc states:

> Our plugins are compatible with IntelliJ 2017.3.3 and later. 

But that's not true. If you look at "Bazel for Android Studio", here: https://plugins.jetbrains.com/plugin/9185-bazel

You'll see a version for `171.4249—171.*` and a version for `181.5281.24.32.4886486—181.*`. In other words, you can have Bazel on Android Studio `171.*` and `181.*`.

Android 3.1.4, released in July is `173.*` and there's no plugin for it.

Reply:
@tbroyer @ronshapiro I don't know the back story there, there's probably a good reason, probably worth asking Bazel and AS teams.

Reply:
In the meantime I'll try 3.2 beta.